apis/github

```diff
@@ -24,12 +27,7 @@ local function writeAuth(data)
 	f.write(textutils.serialize(data))
 	f.close()
 end
-local Auth = {}
-Auth.__index = Auth
-Auth.new = function(type, user, token)
-	return setmetatable({type=type, user=user, token=token}, Auth)
-end
-Auth.getAll = function()
+local function getAuthTable()
```

Comment:
This was moved into a private function for two reasons:

1. I needed Auth.get to return an instance of Auth where as it currently returns just a table.
2. I think it's safer to not expose the entire authTable in an api method.